### ISSUE TITLE
Error on Lighthouse Accessibility Score - Mobile only

### ISSUE THREAD

**Issue by thurfir** [GoogleChrome/lighthouse#15781]
### FAQ

- [X] Yes, my issue is not about [variability](https://github.com/GoogleChrome/lighthouse/blob/main/docs/variability.md) or [throttling](https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md).
- [X] Yes, my issue is not about a specific accessibility audit (file with [axe-core](https://github.com/dequelabs/axe-core) instead).

### URL

https://www.sport-decouverte.com/

### What happened?

Hi, the accessibility score is missing in the Mobile audit when auditing the website, and the background/foreground audit returns an error:

![image](https://github.com/GoogleChrome/lighthouse/assets/22148896/fa13fd2b-90f0-40d1-93db-072aaada3254)

I found a related issue (maybe?) : https://github.com/GoogleChrome/lighthouse/issues/15570 , however we do not use `mix-blend-mode` .

### What did you expect?

The accessibility score to be shown.

### What have you tried?

- Ran lighthouse in Chrome, Pagespeed insights and CLI (side question: is there a way to get more details about the error?).
- Changed font+background colors and sizes on the site to comply with the contrast ratio, and fixed the website to be w3c compliant.
- Tried running axe-core directly in the browser. No error occurred, only a few violations:
![image](https://github.com/GoogleChrome/lighthouse/assets/22148896/6384ba3f-e4b7-4df2-a7d7-9281a0703034)

### How were you running Lighthouse?

CLI

### Lighthouse Version

11.5.0

### Chrome Version

121.0.6167.86

**Comment by S3gillu** [comment]
<img width="1440" alt="Screenshot 2024-01-25 at 4 03 10 PM" src="https://github.com/GoogleChrome/lighthouse/assets/31066063/824f5b40-83fa-4e4a-8805-a788776fe936">

It's the same problem for Desktop Accessibility Results too.

**Comment by thurfir** [comment]
@S3gillu this is strange, I don't have your result:

![image](https://github.com/GoogleChrome/lighthouse/assets/22148896/20f86f31-d48e-44ba-8138-80a0ca1cb577)
https://pagespeed.web.dev/analysis/https-www-sport-decouverte-com/8puw8ykqcn?form_factor=desktop

**Comment by S3gillu** [comment]
@thurfir Please check this link, [https://pagespeed.web.dev/analysis/https-cars24-com-au/rhvlcw5r0h?form_factor=desktop](url)

**Comment by connorjclark** [comment]
fixed by upgrading axe
